include/aws/common/byte_buf.h

```diff
@@ -444,14 +454,19 @@ static inline bool aws_byte_cursor_read_be64(struct aws_byte_cursor *cur, uint64
  * If there is insufficient space in the cursor, returns false, leaving the
  * cursor unchanged.
  */
-static inline bool aws_byte_cursor_write(
-    struct aws_byte_cursor *AWS_RESTRICT cur,
+static inline bool aws_byte_buf_write(
+    struct aws_byte_buf *AWS_RESTRICT cur,
```

Comment:
I see `aws_byte_buf_write()` and `aws_byte_buf_write_from_whole_buffer()`, but there is no `aws_byte_buf_write_from_whole_cursor()`?

Reply:
It's further down in the file.